### PR TITLE
Add negative test case to Delete E2Es.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

**Review by mpichura — APPROVED** [review]
How long did it take you to instruct Copilot to write this tiny test?

**Comment by tpage-alfresco** [comment]
It was very quick to write. I typed in something like `@Test` and then tabbed a few times. Copilot made one mistake - guessing the name of an assertion method, so I had to fix that manually.

**Comment by krdabrowski** [comment]
By "tabbing" you mean picking-up one scenario from a list of possibilities?

File: live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java [hunk 1]
```diff
@@ -66,4 +66,30 @@ void testDeleteRequest()
...
+                {
+                  "specversion": "1.0",
+                  "type": "org.alfresco.event.node.Deleted",
+                  "id": "df329995-d744-427c-bafb-4a31ba7d50e3",
+                  "source": "/08d9b620-48de-4247-8f33-360988d3b19b",
+                  "time": "2021-01-27T10:57:02.586606Z",
+                  "dataschema": "https://api.alfresco.com/schema/event/repo/v1/nodeDeleted",
+                  "datacontenttype": "application/json",
+                  "data": {
+                    "eventGroupId": "acb8e25f-a340-48b5-8de8-249ae5bac670"
```
Comment (krdabrowski) [inline live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java:87]: Is it even possible to receive such a Repo event without a resource?

File: live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java [hunk 2]
```diff
@@ -66,4 +66,30 @@ void testDeleteRequest()
                 }""";
         containerSupport.expectHxIngestMessageReceived(expectedBody);
     }
+
+    @Test
+    void testDeleteRequestWithNoResource()
+    {
+        // given
+        containerSupport.prepareHxInsightToReturnSuccess();
```
Comment (krdabrowski) [inline live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java:74]: Should HxI mock be prepared to return 200 if there shouldn't be any ingestion performed? 🤔

**Comment by tpage-alfresco** [comment]
> By "tabbing" you mean picking-up one scenario from a list of possibilities?

By "tabbing" I mean "pressing tab".

File: live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java (hunk 1 above)
Comment (tpage-alfresco) [reply live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java:87]: No, not really. It's very unlikely anyway.

File: live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java (hunk 2 above)
Comment (tpage-alfresco) [reply live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/DeleteRequestIntegrationTest.java:74]: No - good spot.

**Pull request closed without merging**